subgraphs/loreum-nft/abis/LoreumNFT.json

```diff
@@ -32,7 +27,7 @@
         "type": "address"
       }
     ],
-    "stateMutability": "nonpayable",
```

Comment:
I revised the constructor for `LoreumNFT` contract to be `nonpayable` ... would recommend we stick with that here in the JSON (ABI)

Reply:
Initially this was to allow us to "preMint" 9 NFTs and send in `mintCost` * 9 during construction, but since we haven't added that functionality yet, I ended up removing the `payable` modifier

Reply:
This is just a case of the ABI not being updated because we are not using npm to serve the sontract artifacts.